examples/common/apps.ts

```diff
@@ -6,6 +6,7 @@ const apps = [
   "middleware",
   "vercel-blog-starter",
   "vercel-commerce",
+  "ssg-app",
```

Comment:
I renamed api to playground (and now I realize I missed l.4), maybe you can re-use that playground to add your stuff instead of creating yer another app?

Reply:
playground is not fully SSG:
![Screenshot 2025-02-12 at 20 42 29](https://github.com/user-attachments/assets/1d461de6-b74a-4169-a8a7-96a95e1a0e8a)

I intentionally added a new app that is fully SSG so that we can make sure that such an application can also be successfully run (if I remember correctly there were edge cases in next-on-pages in which an app with only SSG routes could not run)

I do think that this is a valuable addition, but if you strongly disagree I can try to reuse playground

Reply:
> playground is not fully SSG:

I think the important thing is that the layout/page you add are SSG?
Does whatever other existing routes are have any impact?

Reply:
> if I remember correctly there were edge cases in next-on-pages in which an app with only SSG routes could not run

Oh maybe I should have read your comment fully before answering :)

Reply:
> Does whatever other existing routes are have any impact?

yeah probably not 🙂

Reply:
anyways having an only-SSG app might be not needed, personally I think that it is a worthwhile addition that doesn't really hurt, but again if you strongly disagree I can just reuse the playground one

Reply:
No pb if you feel like it's a worthwhile addition.